components/brave_rewards/browser/rewards_service_observer.h

```diff
@@ -61,6 +61,9 @@ class RewardsServiceObserver : public base::CheckedObserver {
   virtual void OnPublisherListNormalized(
       RewardsService* rewards_service,
       brave_rewards::ContentSiteList list) {};
+  virtual void OnRemovePendingContribution(
```

Comment:
same as above `OnPendingContributionRemoved`

Reply:
since this is an observer notification it should have information about the contribution that was removed

Reply:
not sure why this would be needed. No UI needs this info, so we would just pass it along and don't use it